Nova/control/chatbot-services.js

```diff
@@ -36,35 +36,41 @@ const fetchResponse = (userInput) => {
             // GREETINGS
             if (userInput.toLowerCase().includes("hello")) {
```

Comment:
Try to make the chat bot control the conversation, instead of saying "type anything"